beerchat.lua

```diff
@@ -0,0 +1,7 @@
+
+beerchat.register_callback("before_join", function(name, channel)
+        if channel == "mod" then
+                local success = minetest.check_player_privs(name, "staff")
+                return success
```

Comment:
Here's also previous comment as example code, this allows executing rest of before_join callbacks when target channel is #mod.

Not requirement, previous solution should work too. This simply allows continuing with possible other before_join callbacks.
```suggestion
        if channel == "mod" and not minetest.check_player_privs(name, "staff") then
                return false
```